tests/tck/features/match/SeekById.feature

```diff
@@ -128,6 +128,28 @@ Feature: Match seek by id
       | 'Jonathon Simmons' |
       | 'Klay Thompson'    |
       | 'Dejounte Murray'  |
+    When executing query:
```

Comment:
Add tests about #5063 ?

Reply:
ok

Reply:
But this PR don't resolve #5063

Reply:
The pr is not to really resolve #5063, but to handle the bug of `Incorrect match query result`.

Reply:
For 3.4.0, I think it's ok.

Reply:
ok